view/challenge-specific-view.php

```diff
@@ -30,12 +30,12 @@
         <strong><?= $challenge['score'] ?> points</strong>
     </p>
     <p>Location: 
-        <strong><a href="<?= LIST_CHALLENGES_PATH ?>">
+        <strong><a href="<?= $listChallenges_path ?>">
```

Comment:
Are we using a PHP variable for the list challenge path?

Reply:
I've just changed! :) please review once again! @LEMON4DE